compose/snippets/src/main/java/com/example/compose/snippets/glance/GlanceSnippets.kt

```diff
@@ -868,6 +870,34 @@ object GlanceTheming {
     }
 }
 
+object GlanceInnerPadding {
+
+    /**
+     * Applies corner radius for views that are visually positioned [widgetPadding]dp inside of the
+     * widget background.
+     */
+    @Composable
+    fun GlanceModifier.appWidgetInnerCornerRadius(): GlanceModifier {
+        // [START android_compose_glance_innercornerradius]
+        if (Build.VERSION.SDK_INT < 31) {
+            return this
+        }
+
+        // Padding around the widget
+        val widgetPadding = 12.dp
+        val resources = LocalContext.current.resources
+        // get dimension in float (without rounding).
+        val px = resources.getDimension(android.R.dimen.system_app_widget_background_radius)
+        val widgetBackgroundRadiusDpValue = px / resources.displayMetrics.density
```

Comment:
Can you use the helper function:
```
val widgetBackgroundRadiusDpValue =  with(LocalDensity.current){ intValue.toDp() }
```

Reply:
LocalDensity isn't available during a glance composition.  This is a glance snippet so the code would crash.